src/procedures/ForceTransfer.ts

```diff
@@ -43,6 +43,26 @@ export class ForceTransfer extends Procedure<ForceTransferArgs> {
       });
     }
 
+    const controller = await securityToken.controller();
+    let account: string;
+
+    if (currentWallet) {
+      ({ address: account } = currentWallet);
+    } else {
+      throw new PolymathError({
+        message:
+          "No default account set. You must pass token owner's private key to Polymath.connect()",
+        code: ErrorCodes.ProcedureValidationError,
+      });
+    }
+
+    if (account !== controller) {
+      throw new PolymathError({
+        code: ErrorCodes.ProcedureValidationError,
+        message: `Unauthorized`,
```

Comment:
The message should be a bit more specific. Maybe something like `You must be the controller of this Security Token to perform forced transfers. Did you remember to call "setController"?`